demisto_sdk/commands/content_graph/parsers/pack.py

```diff
@@ -325,6 +339,17 @@ def parse_ignored_errors(self):
         """Sets the pack's ignored_errors field."""
         self.ignored_errors_dict = dict(get_pack_ignore_content(self.path.name) or {})  # type: ignore
 
+    def parse_pack_readme(self):
+        """Sets the pack's readme field."""
+        path = f"{self.path}/README.md"
+        try:
+            with open(path) as f:
+                self.pack_readme = f.read()
```

Comment:
```suggestion
                self.pack_readme = f.read()
```

can we call the `parse_pack_readme` from the constructor and assign `self.pack_readme` there? its more readable IMO.

Reply:
I think it makes more sense to put it here because it's the same place we put the `parse_ignored_errors`.